kpatch-build/kpatch-build

```diff
@@ -602,6 +602,13 @@ echo "Building patch module: kpatch-$PATCHNAME.ko"
 cp "$OBJDIR/.config" "$SRCDIR"
 cd "$SRCDIR"
 make prepare >> "$LOGFILE" 2>&1 || die
+
+if [[ $DISTRO == ubuntu ]]; then
+  # UBUNTU: add UTS_UBUNTU_RELEASE_ABI to utsrelease.h after regenerating it
+  UBUNTU_ABI=$(echo "$ARCHVERSION" | awk '{split($1, a, "-"); print a[2]}')
```

Comment:
Ah, I see.  I think doing it in two lines is fine and still would be better than having to spawn processes.